pyro/distributions/transforms/planar.py

```diff
@@ -14,6 +14,11 @@
 
 @copy_docs_from(Transform)
 class ConditionedPlanar(Transform):
+    domain = constraints.real
```

Comment:
we may want to change this to `constraints.real_vector` at some point, but this is fine for now